### PR TITLE
Add describe_parameters support to SSM

### PR DESCRIPTION
**Pull request by declanshanaghy**

### CODE REVIEW

**Comment by JackDanger** [comment]
@declanshanaghy Thanks for the contribution! The new code needs tests to cover it, can you take a shot at implementing the tests and if you get stuck reach out here for help?

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 setup(
     name='moto',
-    version='1.0.1',
+    version='1.0.1.1',
```
Comment (JackDanger) [inline setup.py]: Please leave this at the old version until spulec cuts a new release

Comment (declanshanaghy) [inline setup.py]: ok. was using it to rev our internal forked version

**Comment by JackDanger** [comment]
Nice work, @declanshanaghy! Merging

**Pull request merged**